### DISCUSSION TITLE
Config flow could not be loaded: 500 Internal Server Error Server got itself in trouble

### DISCUSSION THREAD

**Discussion by aleksanderlvr** [DurgNomis-drol/ha_toyota#240] [Q&A]
Since the 6th of February i have been getting this error again. Is it a problem with me or Toyota's servers?

**Comment by CM000n** [comment]
It is caused because of this: https://github.com/DurgNomis-drol/ha_toyota/issues/236

**Reply by aleksanderlvr** [reply]
Thank you for all your work. Can I buy you a coffee somewhere?

**Reply by CM000n** [reply]
Thank you for your kind words. 😊 
But if you really can spare some money, spend it on people who can really use it.

- https://www.unicef.de/spenden
- https://naturerbe.nabu.de/spenden-und-helfen/index.html
